sandbox-search.js

```diff
@@ -11,13 +11,23 @@ export default async function () {
   const text = await readFile('sandbox-seed.json', { encoding: 'utf-8' })
   const { circulars, synonyms } = JSON.parse(text)
 
+  const groupedSynonyms = synonyms.reduce((accumulator, synonym) => {
```

Comment:
I was thinking something more along the lines of https://lodash.com/docs/#groupBy.

Reply:
is there a reason or is it just preference?

Reply:
It's terser and it's more obvious what the code does. I think it is more idiomatic. I would have suggested [`Object.groupBy`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy), but it is a relatively recent addition to ECMAScript and requires a newer version of Node.js than we are targeting.